### ISSUE TITLE
Promoting select RFCs to stable & adapting updated COSS / template

### ISSUE THREAD

**Issue by jimstir** [vacp2p/rfc-index#58]
Reference: https://github.com/vacp2p/rfc/issues/528
Author: kaiserd

This milestone issue tracks the promotion of select RFCs to the `stable` status,
as well as their adaption to the updated [1/COSS](https://rfc.vac.dev/spec/1/) and [RFC template](https://github.com/vacp2p/rfc/blob/master/content/docs/rfcs/template/README.md).

## Blockers

~For now, this issue is blocked, waiting for the `store` flag implementation to be in release versions of at least two Waku implementation, as well as some in-the-wild testing thereafter.
However, many of prerequisite tasks described in this issue can already be started and merged to update draft versions of the respective RFCs. Only the actual promotions to stable are blocked.~

* [x] store flag support for  for [14/WAKU2-MESSAGE](https://rfc.vac.dev/spec/14/) in nwaku release version
* [x] store flag support  for [14/WAKU2-MESSAGE](https://rfc.vac.dev/spec/14/) in the release version of one further Waku implementation
* [x] wait until store flag support has been successful for some time in the wild

## Background

The protocols specified in the following RFCs are ready for the `stable` status

* [10/WAKU2](https://rfc.vac.dev/spec/10/)
* [13/WAKU2-STORE](https://rfc.vac.dev/spec/13/)
* [14/WAKU2-MESSAGE](https://rfc.vac.dev/spec/14/)
* [15/WAKU-BRIDGE](https://rfc.vac.dev/spec/15/)

This milestone issue tracks the tasks necessary for promoting these RFCs to stable.
These tasks comprise

* thorough proofreading for ambiguity or lack of clarity, as well as the respective editing
* adding a `category` as specified in the updated [1/COSS](https://rfc.vac.dev/spec/1/)
* adding tags as specified in the updated [1/COSS](https://rfc.vac.dev/spec/1/)
* adding/ordering sections as proposed in our [RFC template](https://github.com/vacp2p/rfc/blob/master/content/docs/rfcs/template/README.md)
  - Theory / Semantics (can be named differently if appropriate)
  - Wireformat / Syntax (can be named differently if appropriate)
  - Security Considerations

~In addition to these tasks, [14/WAKU2-MESSAGE](https://rfc.vac.dev/spec/14/) should be augmented by a `store` flag indicating whether a message should be stored or not. As all other RFCs listed above depend on message, this should be done first.~

- #532
- #551

~Further [21/WAKU2-FAULT-TOLERANT-STORE](https://rfc.vac.dev/spec/21/)'s wire specification should be merged into [13/WAKU2-STORE](https://rfc.vac.dev/spec/13/)
so that RFC 13 reflects what is actually being used. This is important for promoting RFC 13 to stable.~

- #552 

Promoting [10/WAKU2](https://rfc.vac.dev/spec/10/) to stable requires factoring out parts that depend on non-stable RFCs.
These parts will be covered in a new RFC.

## Action Items / Issues

* [x] add `ephemeral` flag to  [14/WAKU2-MESSAGE](https://rfc.vac.dev/spec/14/)
* [x] #552
* [x] #551 
* [ ] RFC 15: promote to stable & update to new structure
* [ ] RFC 10: promote to stable & update to new structure
  - promotion to stable includes factoring out parts that depend on non-stable RFCs
* [ ] new RFC ??/WAKU2-unstable: covers parts of original RFC 10 that depended on non-stable RFCs
